libtiledbsoma/src/soma/column_buffer.h

```diff
@@ -128,9 +127,7 @@ class ColumnBuffer {
     ColumnBuffer(const ColumnBuffer&) = delete;
     ColumnBuffer(ColumnBuffer&&) = default;
 
-    ~ColumnBuffer() {
-        LOG_TRACE(fmt::format("[ColumnBuffer] release '{}'", name_));
-    }
+    ~ColumnBuffer();
```

Comment:
Why?  While `trace` is super-voluminous it can be helpful to see that dtors have been reached.

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/1852/files#diff-e563ff857d3a241fa155b2c78fdd243bbe474e8bf28a7782d03f56f00d00f6b5R156-R158

It's because of `fmt::format`. I've moved it into `column_buffer.cc`, so we still do a get a log trace.